shared/src/vyxal/Elements.scala

```diff
@@ -3928,6 +3928,20 @@ object Elements:
     ) {
       case a: VNum => a / VNum(spire.math.Real.pi) * VNum(180)
     },
+    addPart(
+      Dyad,
+      "ÞR",
+      "Reshape",
+      List("reshape"),
+      false,
+      "a: lst, b: lst[num] => a reshaped to shape b",
+    ) {
+      case (a, b: VList) if b.forall(_.isInstanceOf[VNum]) =>
+        ListHelpers
+          .reshape(ListHelpers.makeIterable(a), b.map(_.asInstanceOf[VNum]))
```

Comment:
I think maybe we could accept any kind of lists instead of having a guard, and only check the shape later so we could tell the user which particular element was not a number
```suggestion
      case (a, b: VList) =>
        val shape = b.map {
          case n: VNum => n.toInt
          case x => throw element x is not an int expectedo nly ints for reshape or something
        }.toList // To force it, idk if there's a better way to force generically (LazyList does have a `force` method but it's not on Seq)
        ListHelpers
          .reshape(ListHelpers.makeIterable(a), shape)
```